<!DOCTYPE html>
<html><head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="/css/style.css">
    <link rel="preload" href="icons.svg" as="image">
    
    
    <title>O.B. | BioInf CookBook</title>
</head>
<body>
        <div class="container"><div id="nav-panel" class="row">
    <hr class="topline"/>
        <input type="checkbox" id="checkbox">
        <label for="checkbox" class="menu-toggle">
            <svg viewBox="0 0 25 25"><use xlink:href="/icons.svg#hamburger"></use></svg>
        </label>
        <label for="checkbox" class="menu-toggle-text">
            O.B. &mdash; Site menu
        </label>
        <nav>
            <ul id="nav-menu">
                
                <li class="menu-item"><a class="nav-link" href="/">
                    
                    Home
                </a></li>
                
                <li class="menu-item"><a class="nav-link" href="/research/">
                    
                    Research
                </a></li>
                
                <li class="menu-item"><a class="nav-link" href="/students/">
                    
                    Students
                </a></li>
                
                <li class="menu-item"><a class="nav-link" href="/teaching/">
                    
                    Teaching
                </a></li>
                
                <li class="menu-item"><a class="nav-link" href="/cookbook/">
                    
                    BioInf CookBook
                </a></li>
                
                <li class="menu-item"><a class="nav-link" href="/contacts/">
                    
                    Contact
                </a></li>
                
                <li class="menu-item"> ~ </li>
            <li class="menu-item">
            <a class="nav-link nav-icon-link" href="mailto:olga.bochkareva@ist.ac.at">
                <svg viewBox="0 0 25 25"><use xlink:href="/icons.svg#i-envelope"></use></svg>
            </a>
            
            <a class="nav-link nav-icon-link" href="https://twitter.com/o_bochkareva">
                <svg viewBox="0 0 25 25"><use xlink:href="/icons.svg#i-twitter"></use></svg>
            </a>
            </li></ul>
        </nav>
</div>
<br/>

                <div id="table-of-contents">
                    <h2> Index, <a href="#" class="nav-link">top</a> </h2> 
                    <div id="text-table-of-contents" class="not-a-text-link">
                        <nav id="TableOfContents">
  <ul>
    <li><a href="#assemblers">assemblers</a></li>
    <li><a href="#genome-annotation">genome annotation</a></li>
    <li><a href="#alignments">alignments</a></li>
    <li><a href="#orthologs">orthologs</a></li>
    <li><a href="#pangenomics">pangenomics</a></li>
    <li><a href="#whole-genome-alignment">whole-genome alignment</a></li>
    <li><a href="#rearrangements">rearrangements</a></li>
    <li><a href="#phylogenetics">phylogenetics</a></li>
    <li><a href="#horizontal-gene-transfer-hgt">horizontal gene transfer (HGT)</a></li>
    <li><a href="#ancestral-reconstruction">ancestral reconstruction</a></li>
    <li><a href="#visualization">visualization</a></li>
    <li><a href="#datasets">datasets</a></li>
  </ul>
</nav>
                    </div>
                </div>
            
            <div id="content">
<div class="main-text">
    <h1>BioInf CookBook</h1>
    <p>Tools, packages and software for bacterial genomics.</p>
<p>The initiative was started in Gelfand lab. I thank all colleagues for their suggetions and comments.</p>
<h2 id="assemblers">assemblers</h2>
<p><strong>üîß Tool</strong> <a href="https://github.com/ablab/spades">SPAdes assembler</a> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3342519/">paper</a>) - an assembly toolkit containing various assembly pipelines.</p>
<!-- ## genome browser -->
<h2 id="genome-annotation">genome annotation</h2>
<p><strong>üåç WebTool</strong> <a href="https://kbase.us/applist/apps/RAST_SDK/reannotate_microbial_genome/release?gclid=Cj0KCQjw59n8BRD2ARIsAAmgPmI2nvh6bDU_OkagT76hMuxDqv7jBRq2GXqc1u3LZqa-cwpUuOqcfrEaAvdUEALw_wcB">Rast</a> (<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-9-75">paper</a>) - a fully automated service for annotating bacterial and archaeal genomes. The service identifies protein-encoding, rRNA and tRNA genes, assigns functions to the genes, predicts which subsystems are represented in the genome, uses this information to reconstruct the metabolic network and makes the output easily downloadable for the user.</p>
<p><strong>üîß Tool</strong> <a href="https://github.com/tseemann/prokka">Prokka</a> (<a href="https://academic.oup.com/bioinformatics/article/30/14/2068/2390517">paper</a>) - a command line software tool that can be installed on any Unix system. Prokka coordinates a suite of existing software tools to achieve a rich and reliable annotation of genomic bacterial sequences. Where possible, it will exploit multiple processing cores, and a typical bacterial genome can be annotated in ‚àº10 min on a quad core desktop computer. It is well suited to iterative models of sequence analysis and integration into genomic software pipelines.</p>
<p><strong>üîß Tool</strong> <a href="https://github.com/hyattpd/Prodigal">Prodigal</a> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2848648/">paper</a>) - a fast, lightweight, open source gene prediction program, focused specifically on the three goals of improved gene structure prediction, improved translation initiation site recognition, and reduced false positives.</p>
<p><strong>üåç WebTool</strong> <a href="https://papers.genomics.lbl.gov/cgi-bin/litSearch.cgi">PaperBLAST</a> (<a href="https://msystems.asm.org/content/2/4/e00039-17">paper</a>)  -  web tool uses EuropePMC to search the full text of scientific articles for references to genes. PaperBLAST also takes advantage of curated resources (Swiss-Prot, GeneRIF, and EcoCyc) that link protein sequences to scientific articles. PaperBLAST‚Äôs database includes over 700,000 scientific articles that mention over 400,000 different proteins. Given a protein of interest, PaperBLAST quickly finds similar proteins that are discussed in the literature and presents snippets of text from relevant articles or from the curators.</p>
<!-- ## databases -->
<h2 id="alignments">alignments</h2>
<p><strong>üåç WebTool</strong> <a href="http://tcoffee.crg.cat/">T-Coffee</a> (<a href="http://tcoffee.crg.cat/">paper</a>) - a web server for the multiple sequence alignment of protein and RNA sequences using structural information and homology extension, includes different alignment models, such as MAFFT, MUSCLE and Kalign. Pro-Coffee is suiatable for alignment of promoter regions.</p>
<p><strong>üîß Tool</strong> <a href="https://mafft.cbrc.jp/alignment/software/">MAFFT-DASH</a> (<a href="https://academic.oup.com/nar/article/47/W1/W5/5486273">paper</a>) - a web server that integrates structural alignments with the MAFFT multiple sequence alignment (MSA) tool.</p>
<p><strong>üîß Tool</strong> <a href="https://bioweb.supagro.inra.fr/macse/">MACSE</a> - a tool aligns coding NT sequences with respect to their AA translation while allowing NT sequences to contain multiple frameshifts and/or stop codons. MACSE is hence the first automatic solution to align protein-coding gene datasets containing non-functional sequences (pseudogenes) without disrupting the underlying codon structure. It has also proved useful in detecting undocumented frameshifts in public database sequences and in aligning next-generation sequencing reads/contigs against a reference coding sequence.</p>
<h2 id="orthologs">orthologs</h2>
<p><strong>üîß Tool</strong> <a href="https://www.bioinf.uni-leipzig.de/Software/proteinortho/">ProteinOrtho</a> (<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-124">paper</a>) - a tool to detect orthologous genes within different species. For doing so, it compares similarities of given gene sequences and clusters them to find significant groups. The algorithm was designed to handle large-scale data and can be applied to hundreds of species at one.</p>
<p><strong>üíæ Data</strong> <a href="https://omabrowser.org/oma/home/">OMA orthology database</a> (<a href="https://academic.oup.com/bioinformatics/article/33/14/i75/3953943">paper</a>)</p>
<h2 id="pangenomics">pangenomics</h2>
<p><strong>üìÑ Review</strong> Costa, S√°vio &amp; Guimar√£es, Lu√≠s &amp; Silva, Artur &amp; Soares, Siomar &amp; Bara√∫na, Rafael. (2020). <a href="https://journals.sagepub.com/doi/full/10.1177/1177932220938064">First Steps in the Analysis of Prokaryotic Pan-Genomes</a>. Bioinformatics and Biology Insights. 14. 117793222093806. 10.1177/1177932220938064</p>
<p><strong>üìñ Book</strong> The Pangenome: Diversity, Dynamics and Evolution of Genomes. Editors: Tettelin, Herv√©, Medini, Duccio (Eds.) (2020)</p>
<p><strong>üîß Tool</strong> <a href="https://sanger-pathogens.github.io/Roary/">Roary</a> (<a href="https://academic.oup.com/bioinformatics/article/31/22/3691/240757">paper</a>) - a tool that rapidly builds large-scale pan genomes, identifying the core and accessory genes. Roary makes construction of the pan genome of thousands of prokaryote samples possible on a standard desktop without compromising on the accuracy of results.</p>
<p><strong>üîß Tool</strong> <a href="https://github.com/harry-thorpe/piggy">Piggy</a> (<a href="https://academic.oup.com/gigascience/article/7/4/giy015/4919733">paper</a>) - a novel pipeline that emulates Roary except that it is based only on intergenic regions. A key utility provided by Piggy is the detection of highly divergent (‚Äúswitched‚Äù) intergenic regions (IGRs) upstream of genes.</p>
<p><strong>üîß Tool</strong> <a href="https://github.com/gem-pasteur/PanACoTA">PanACoTA</a> (<a href="https://www.biorxiv.org/content/10.1101/2020.09.11.293472v3">paper</a>) - a tool that allows to download all genomes of a species, build a database with those passing quality and redundancy controls, define uniform annotation, and use them to build a pangenome, several variants of core or persistent genomes, their alignments, and a rapid but accurate phylogenetic tree. While many programs have become available in the last few years to build pangenomes, we have focused on a method that tackles all the key steps of the process, from download to phylogenetic inference.</p>
<h2 id="whole-genome-alignment">whole-genome alignment</h2>
<p><strong>üîß Tool</strong> <a href="%5Bpaper%5D(https://www.nature.com/articles/s41467-020-19777-8)">SibeliaZ</a> - an algorithm for identifying collinear blocks in closely related genomes based on analysis of the de Bruijn graph, incorporated into a multiple whole-genome alignment pipeline. Instruction how to construct synteny blocks with given length threshold <a href="https://github.com/ctlab/parallel-rearrangements/blob/master/BLOCKS-OBTAIN.md">here</a></p>
<p><strong>üîß Tool</strong> <a href="http://darlinglab.org/mauve/mauve.html">ProgressiveMauve</a> (<a href="https://www.ncbi.nlm.nih.gov/pubmed/20593022">paper</a>) is a system for constructing multiple genome alignments in the presence of large-scale evolutionary events such as rearrangement and inversion. The alignment display is organized into one horizontal ‚Äúpanel‚Äù per input genome sequence. Each genome‚Äôs panel contains the name of the genome sequence, a scale showing the sequence coordinates for that genome, and a single black horizontal center line. Colored block outlines appear above and possibly below the center line. Each of these block outlines surrounds a region of the genome sequence that aligned to part of another genome, and is presumably homologous and internally free from genomic rearrangement. However, the algorithm‚Äôs compute time scales cubically in the number of genomes to align, making it unsuitable for datasets containing more than 50-100 bacterial genomes.</p>
<h2 id="rearrangements">rearrangements</h2>
<p><strong>üìÑ Review</strong> Elise Darmon, David R. F. Leach <a href="https://mmbr.asm.org/content/78/1/1">Bacterial Genome Instability</a> Microbiology and Molecular Biology Reviews Mar 2014, 78 (1) 1-39</p>
<p><strong>üîß Tool</strong> <a href="https://github.com/jenniferlu717/SkewIT">SkewIT</a> (<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008439">paper</a>) - a tool for analyzing GC Skew in bacterial genomes. GC Skew is a phenomenon observed in many bacterial genomes wherein the two strands of the chromosome contain different proportions of guanine/cytosine nucleotides. SkewIT quantifies GC Skew using a single metric that can then be compared/analyzed across thousands of bacterial genomes.</p>
<p><strong>üåç WebTool</strong> <a href="https://bioinfo.lifl.fr/yass/yass.php">YASS</a> (<a href="https://nar.oxfordjournals.org/cgi/content/abstract/33/suppl_2/W540">paper</a>) - a DNA local alignment tool based on an efficient and sensitive filtering algorithm. It applies transition-constrained seeds to specify the most probable conserved motifs between homologous sequences, combined with a flexible hit criterion used to identify groups of seeds that are likely to exhibit significant alignments.</p>
<p><strong>üîß Tool</strong> <a href="https://pypi.org/project/PaReBrick/">PAREBRICK</a></p>
<h2 id="phylogenetics">phylogenetics</h2>
<p><strong>üìÑ Review</strong> Kapli et al. <a href="https://pubmed.ncbi.nlm.nih.gov/32424311/">Phylogenetic tree building in the genomic age</a></p>
<p><strong>üîß Tool</strong> <a href="https://github.com/stamatak/standard-RAxML">RAxML</a> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3998144/">paper</a>) - a popular program for phylogenetic analyses of large datasets under maximum likelihood with substantial extension of substitution models and supported data types, the introduction of SSE3, AVX and AVX2 vector intrinsics, techniques for reducing the memory requirements of the code and a plethora of operations for conducting post-analyses on sets of trees.</p>
<p><strong>üîß Tool</strong> <a href="https://github.com/NBISweden/MrBayes/tree/v3.2.7a">MrBayes</a> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3329765/">paper</a>) - a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. MrBayes uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.</p>
<p><strong>üîß Tool</strong> <a href="https://github.com/LANL-Bioinformatics/PhaME">PhaME</a> (<a href="https://www.nature.com/articles/s41598-020-58356-1">paper</a>) - phylogenetic and molecular evolutionary analysis from sequencing reads, draft assemblies or completed genomes of closely related organisms. Rapid phylogenetic characterization of organisms within complex samples without the need for prior assembly.</p>
<p><strong>üì¶ Python Package</strong> <a href="http://etetoolkit.org/">ete3</a> a tool for comparison of phylogenetic trees.</p>
<p><strong>üîß Tool</strong> <a href="https://bioweb.supagro.inra.fr/macse/">TreeCmp</a> <a href="https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13358">paper</a> - a package of applications for comparing phylogenetic tree sets. Visual TreeCmp includes a graphical web interface allowing the visualization of compared trees and command line application extended by comparison methods recently proposed in the literature.</p>
<p><strong>üåç WebTool</strong> <a href="https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13358">iTOL</a> - an online tool for the display, annotation and management of phylogenetic trees. Allows to explore your trees directly in the browser and annotate them with various types of data.</p>
<p><strong>üîß Tool</strong> <a href="https://git.scicore.unibas.ch/TBRU/Treemmer">Treemmer</a> (<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2164-8">paper</a>) - a tool to reduce large phylogenetic datasets with minimal loss of diversity. Treemmer can reduce the size of datasets with different phylogenetic structures and levels of redundancy while maintaining a sub-sample that is representative of the original diversity. Additionally, it is possible to fine-tune the behavior of Treemmer including any kind of meta-information, making Treemmer particularly useful for empirical studies.</p>
<h2 id="horizontal-gene-transfer-hgt">horizontal gene transfer (HGT)</h2>
<p><strong>üîß Tool</strong> <a href="https://shadowcaster.readthedocs.io/en/latest/">ShadowCaster</a> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7397055/">paper</a>) - a hybrid approach that sequentially combines nucleotide composition-based predictions by support vector machines (SVMs) under the shadow of phylogenetic models independent of tree reconstruction, to improve the detection of HGT events in prokaryotes. ShadowCaster successfully predicted close and distant HGT events in bacterial genomes. Requires a lot of components (R, Perl, OrthoMCL, Blastp and EMBOSS package).</p>
<p><strong>üîß Tool</strong> <a href="http://www.helsinki.fi/bsg/software/BRAT-NextGen/">BratNextGen</a>(<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245952/">paper</a>) - a method for detecting homologous recombination events from whole-genome sequence data for bacterial population samples on a large scale. The approach can efficiently handle hundreds of whole genome sequenced population samples and identify separate origins of the recombinant sequence, offering an enhanced insight into the diversification of bacterial clones at the level of the whole genome.</p>
<h2 id="ancestral-reconstruction">ancestral reconstruction</h2>
<p><strong>üîß Tool</strong> <a href="https://github.com/NBISweden/MrBayes/tree/v3.2.7a">MrBayes</a> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3329765/">paper</a>) - a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. MrBayes uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.</p>
<p><strong>üîß Tool</strong> <a href="http://abacus.gene.ucl.ac.uk/software/paml.html">PAML</a> (<a href="https://academic.oup.com/mbe/article/24/8/1586/1103731">paper</a>) - a package of programs for phylogenetic analyses of DNA and protein sequences using maximum likelihood (ML). The programs may be used to compare and test phylogenetic trees, but their main strengths lie in the rich repertoire of evolutionary models implemented, which can be used to estimate parameters in models of sequence evolution and to test interesting biological hypotheses.</p>
<h2 id="visualization">visualization</h2>
<p><strong>üåç WebTool</strong> <a href="https://itol.embl.de/">iTOL</a> (<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987883/">paper</a>) - an online tool for the display, annotation and management of phylogenetic trees. Allows to explore your trees directly in the browser and annotate them with various types of data.</p>
<p><strong>üåç WebTool</strong> <a href="https://github.com/jameshadfield/phandango/wiki">Phandango</a> (<a href="https://academic.oup.com/bioinformatics/article/34/2/292/4212949">paper</a>) - a fully interactive tool to allow visualisation of a phylogenetic tree, associated metadata and genomic information such as recombination blocks, pan-genome contents or GWAS results.</p>
<p><strong>üîß Tool</strong> <a href="https://katlabs.cc/genegraphics/">Gene Graphics</a> (<a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx793">paper</a>) - an application that allows for consistent, visually appealing representations of physical gene neighborhoods with minimal effort or expertise.</p>
<p><strong>üåç WebTool</strong> <a href="https://biorender.com/">Biorender</a> - a tool to help scientists create and share professional scientific figures.</p>
<p><strong>üåç WebTool</strong> <a href="https://www.overleaf.com/">Overleaf</a> - a collaborative cloud-based LaTeX editor used for writing, editing and publishing scientific documents. It partners with a wide range of scientific publishers to provide official journal LaTeX templates, and direct submission links.</p>
<h2 id="datasets">datasets</h2>
<p><strong>üíæ Data</strong> <a href="https://www.nature.com/articles/s41587-020-0603-3">A unified catalog of 204,938 reference genomes from the human gut microbiome</a>.</p>
<p><strong>üíæ Data</strong> <a href="https://github.com/zseferbekova/ShigellaProject">genomic data of 414 E.coli/Shigella genomes (full assemblies)</a>. The project contains the phylogenetic tree, alignments of common orthogroups, synteny blocks. <a href="https://www.biorxiv.org/content/10.1101/2020.06.12.147751v1.abstract">Citation</a></p>

</div>
<br/>

            </div>
        </div>
        <footer class="footer"><div class="site-footer">
    <p class="footer-text nav-link">Copyright (c) 2018&mdash;2021 <b><a class="nav-link" href="/">Olga Bochkareva</a></b>
    </p>
</div>
</footer>
    </body>
</html>
